tests/test_repository.py

```diff
@@ -13,7 +13,7 @@
 import responses
 from dateutil.relativedelta import relativedelta
 
-from bugbug import repository
+from bugbug import repository, utils
```

Comment:
Need to move the ExpQueue tests to test_utils.py, as test_repository.py is only meant to test repository.py functions.

Reply:
Of course. I'll push an update tomorrow.

Reply:
done